site/app/libraries/database/DatabaseQueries.php

```diff
@@ -1744,6 +1744,11 @@ public function getRotatingSectionsForGradeableAndUser($g_id, $user_id = null) {
     public function getUsersByRotatingSections($sections, $orderBy = "rotating_section") {
         $return = array();
         if (count($sections) > 0) {
+            $orderBy = str_replace(
+                "Peer",
+                1,
+                $orderBy
+            );
```

Comment:
Why was this necessary? Where this function being called that the `$orderBy` parameter has the word "Peer" in it?

Reply:
On line 526 of ElectronicGraderController.php. For team peer graders, we named the rotating section Peer so that it could be differentiated from non-peer rotating sections. There are other ways we could differentiate that I am working on

Reply:
Is that change in some future PR that you and @emaicus are working on as line 526 of ElectronicGraderController.php is just this:
https://github.com/Submitty/Submitty/blob/c9873ecbe67a9129f62450ed134034e60dd72902/site/app/controllers/grading/ElectronicGraderController.php#L526

where there is no second parameter to the function call...